src/__tests__/routes/users/resetPassword.js

```diff
@@ -0,0 +1,173 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import moment from 'moment';
+import app from '../../../index';
+import models, { sequelize } from '../../../models';
+import { sendResetMail, sendSignupMail } from '../../../services/sendMail';
+
+const { expect } = chai;
+chai.use(chaiHttp);
+chai.use(sinonChai);
+
+const apiVersion = '/api/v1';
+const forgotPasswordURL = `${apiVersion}/users/passwords/forgot`;
+const resetPasswordURL = `${apiVersion}/users/passwords/reset`;
+const validId = 2;
+const resetToken = '12ererfbuib23iub328o7rg8hbiuva';
+
+// Create table and seed database
+const seedTestDb = async () => {
+    await models.User.create({
+        email: 'youremail3@andela.com',
+        role: 'passenger'
+    });
+
+    await models.User.create({
+        email: 'youremail4@andela.com',
+        role: 'passenger'
+    });
+
+    await models.Login.create({
+        email: 'youremail3@andela.com',
+        password: 'password'
+    });
+
+    await models.Login.create({
+        email: 'youremail4@andela.com',
+        password: 'password'
+    });
+
+    await models.Reset.create({
+        email: 'youremail3@andela.com',
+        expireTime: new Date(),
+        resetToken: '$2a$10$Yc4fNidn3ih0Z0wRajFhq.AwneQLYR2RWWYQT7PGJdJj4UN1BGJ1K'
+    }),
+
+    await models.Reset.create({
+        email: 'youremail4@andela.com',
+        expireTime: moment
+            .utc()
+            .add(process.env.TOKENEXPIRY, 'seconds')
+            .toLocaleString(),
+        resetToken: '$2a$10$Yc4fNidn3ih0Z0wRajFhq.AwneQLYR2RWWYQT7PGJdJj4UN1BGJ1K'
+    });
+};
+
+// Clear tables of seed
+const clearTestDb = async () => {
+    try {
+        await models['User'].destroy({ where: {}, force: true });
+        await models['Login'].destroy({ where: {}, force: true });
+        await models['Reset'].destroy({ where: {}, force: true });
+    } catch (err) {
+        throw err;
+    }
+};
+
+// clear database and seed data before test
+before(async () => {
+    try {
+        return sequelize.sync({ force: false }).then(async () => {
```

Comment:
Why are you still dropping the table? I thought we spoke about this, it is an expensive operation, we should rather clear the table.

Reply:
I removed the sequelize.sync({ force: true }) in the after hook believing it is the table drop you were referring to. I just read the sequelize docs now that with force:false : false if the table exists, any field additions/modifications/deletions you have won't be executed. So i am removing it in my next push. Thanks